### PR TITLE
added OS version, added ArangoDB version for Windows

### PR DESCRIPTION
**Pull request by fceller**
@Simran-B feel free to improve wording

### CODE REVIEW

File: snippets/3.4/windows.html.in [hunk 1]
```diff
@@ -2,7 +2,6 @@
   <p>
     The following packages are for Windows 7, Windows 10.1, Windows 2008 R2 and higher.
```
Comment (KVS85) [inline snippets/3.4/windows.html.in]: Windows 10.1 is unoffical naming. Also, it's 1809 build while I'm on 1803 and it works fine. Also, Windows 7 and Windows 2008 R2 could be specialized.

I propose the following: `Windows 7 Service Pack 1, Windows 10 (1803 and higher), Windows Server 2008 R2 and higher`.

Comment (dothebart) [reply snippets/3.4/windows.html.in]: like that.

Comment (Simran-B) [reply snippets/3.4/windows.html.in]: @KVS85 I like your proposal.
So Windows 7 without Service Packs doesn't work? Do you happen to know the reason?

Comment (KVS85) [reply snippets/3.4/windows.html.in]: Windows 7 without SP1 is not supported by Microsoft: https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet. I suppose, we won't too.

**Comment by fceller** [comment]
> Adjusted some minor things. What about Nehalem as minimum CPU architecture / requirement for SSE 4.2? Should this also be noted here for 3.4 and above?

Good point. That is indeed something that can easily be overlooked.

> 
> Is it necessary to backport the changes to https://github.com/arangodb/arangodb.com-html-snippets or is that repo obsolete?

It is kind of obsolescent.

**Comment by fceller** [comment]
@Simran-B added remark about Nehalem.

**Pull request merged**